drivers/classic/src/get_global_param.c

```diff
@@ -981,23 +1119,25 @@ get_global_param(FILE *gp)
         else if (global_param.endmonth > MONTHS_PER_YEAR) {
             log_err("The specified simulation end month (%hu) < 0.  Make sure "
                     "that the global file defines a positive integer for "
-                    "ENDMONTH.", global_param.endmonth);
+                    "ENDMONTH.",
+                    global_param.endmonth);
         }
         if (global_param.endday == 0) {
             log_err("Simulation end day has not been defined.  Make sure "
                     "that the global file defines ENDDAY.");
         }
-        else if (global_param.endday > lastday[global_param.endmonth]) {
+        else if (global_param.endday > lastday[global_param.endmonth - 1]) {
             log_err("The specified simulation end day (%hu) > the number of "
                     "days in the ENDMONTH (%hu).  Make sure that the global "
                     "file defines a positive integer for ENDDAY.",
```

Comment:
Why the `-1`? Has the range of `global_param.endmonth` changed from `[0-11]` to `[1-12]`?